app/src/main/java/com/matzip/presentation/module/NetworkModule.kt

```
@@ -0,0 +1,73 @@
+
+    @Provides
+    @Singleton
+    fun provideLoggingInterceptor(): HttpLoggingInterceptor {
+        val loggingInterceptor = HttpLoggingInterceptor { message ->
+            when {
+                !message.isJsonObject() && !message.isJsonArray() ->
+                    Log.d("RETROFIT1","CONNECTION INFO -> $message")
+                else ->  try {
+                    Log.d("RETROFIT2", GsonBuilder().setPrettyPrinting().create().toJson(
+                        JsonParser().parse(message)))
+                } catch (m: JsonSyntaxException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)